ignition-server/controllers/local_ignitionprovider.go

```diff
@@ -235,6 +236,27 @@ func (p *LocalIgnitionProvider) GetPayload(ctx context.Context, releaseImage str
 		return nil, fmt.Errorf("failed to extract templates from image: %w", err)
 	}
 
+	// Write out the feature gate manifest to the MCC dir.
+	if err := func() error {
+		featureGate := &configv1.FeatureGate{}
+		if err := p.Client.Get(ctx, client.ObjectKey{Name: "cluster"}, featureGate); err != nil {
```

Comment:
Ahh gotcha, wasn't sure which client I had there, will rework

Reply:
Also the way this is right now, there's no trigger event / listener relation here, and this would fetch whatever happen to be there at any point in time. I think this might be good enough for starters, in future we might want to consider the feature gate to be part of the payload `version + config + pull secret signature` and let it be a trigger for generating new payload appropiately.